les/fetcher.go

```diff
@@ -209,6 +212,7 @@ func (f *lightFetcher) syncLoop() {
 			f.checkSyncedHeaders(p)
 			f.syncing = false
 			f.lock.Unlock()
+			f.requestChn <- false
```

Comment:
Why should we emit a loop request here?

Reply:
While syncing we disregard loop requests initiated by new announcements. When we are finished syncing we might have missed a newer head announcement. I've seen this happen. Syncing finished but the client was one block behind the actual head and did not update until another block came. Initiating an extra check after processing new announcements has been re-enabled should fix this scenario.